### ISSUE TITLE
[Package Request]: Python py.exe launcher

### ISSUE THREAD

**Issue by zooba** [microsoft/winget-pkgs#121195]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.python.org/downloads/
Publisher: Python Software Foundation
Package Name: Python launcher
Description: Provides the 'py.exe' launcher
Package Version: 3.11.5
Installer URL: https://www.python.org/ftp/python/3.11.5/win32/launcher.msi

This is normally installed as part of CPython, but can also be installed separately (on purpose, just not widely advertised). The launcher is able to detect Windows Store installs (and to install them via winget), as well as any other properly registered Python distribution, so it can be very useful for people who use those installs but do not want a regular CPython install.

While the version number matches the version of CPython it is bundled with, it does not install in parallel like CPython does, and in general the latest release version is all that should be made available (e.g. when 3.12.0 releases in a couple of weeks, nobody should need any 3.11 versions of the launcher anymore). The URL can be inferred by taking any CPython release URL (the ".exe" file) and replacing the filename with `win32/launcher.msi`. It is only ever released as a 32-bit executable.

The installer defaults to per-user, which is generally fine, but if it is run elevated and `ALLUSERS=1` passed to msiexec it will install for all users.

**Comment by CoolPlayLin** [comment]
@zooba Btw, do you think this software is necessary for python?😊

**Comment by zooba** [comment]
The existing Python packages (that run the installer) already install it, so there's no need. This is to be a standalone install for those who don't want a particular runtime but want the launcher.

Incidentally, version 3.12.0 just released (minutes ago). There's no reason to prefer 3.11.5 over 3.12.0.

**Comment by Dragon1573** [comment]
> Incidentally, version 3.12.0 just released (minutes ago). There's no reason to prefer 3.11.5 over 3.12.0.

Okay ...

According to [Python Releases for Windows](https://www.python.org/downloads/windows/), both `3.12.0` and `3.11.6` was released. I think pull request https://github.com/microsoft/winget-pkgs/pull/121447 can be closed without merge. I'll create brand new PR for `3.12.0`.

Does launcher for `3.12.0` support previous version (such as `3.11.6`, `3.10.3`, `3.9.8`, `3.8.18`)? Shall I create manifests for launcher version `3.11.6`?
